archive/tar_test.go

```diff
@@ -32,6 +33,7 @@ import (
 
```

Comment:
could we remove the empty here? since we don't need to separate the packages into 3 groups 😄

Reply:
Best done on master; this is only a backport of the fix, and I'd rather not make those changes in the release branch; master also has it split into 3 groups (which was done in the original PR adding these); https://github.com/containerd/containerd/commit/0deba01621f2f2399e148a0d072d960526623cc2

Reply:
make senses! thanks.